docker-compose.yml

```diff
@@ -356,7 +356,9 @@ services:
       ofelia.job-exec.sn_channel_cron.schedule: "@every 1m"
       ofelia.job-exec.sn_channel_cron.command: >
         su lnd -c bash -c "
-          if [ $$(lncli getinfo | jq '.num_active_channels + .num_pending_channels') -ge 3 ]; then
+          if [ $$(lncli getinfo | jq '.num_active_channels + .num_pending_channels') -ge 6 ]; then
+            exit 0
```

Comment:
nitpick: Do we really still need this check? You mentioned this:

> This modification keeps the useful check for .active + .pending channels and sets the max to 6. This is useful to prevent endless channels opens that get stuck in pending.

But sn_lnd only opens channel to router_lnd so the check below will make sure we stop after three channels, no?

Reply:
Yes good catch, but I think we still want that check.

This is because `lncli listchannels` does not show pending channels. So the `lncli getinfo` still gives the us that information which is a valuable check to prevent endless "WhileLoop" style channel opens if channels opens moving from pending -> active start taking more than the one minute between the scheduled channel open jobs.

Reply:
Ah ok, makes sense!